### ISSUE TITLE
No docs or README for http package

### ISSUE THREAD

**Issue by DartBot** [dart-lang/http_server#3]
<a href="https://github.com/sethladd"><img src="https://avatars.githubusercontent.com/u/5479?v=3" align="left" width="96" height="96"hspace="10"></img></a> **Issue by [sethladd](https://github.com/sethladd)**
_Originally opened as dart-lang/sdk#9557_

---

I can't find any docs or a README for http package.

Many of our pkg packages are in http://api.dartlang.org/docs/releases/latest/ but I can't find it there.

Many pub packages have a README with simple instructions. http package doesn't have a README, or if does, pub.dartlang.org isn't picking it up :(

This package is awesome, hopefully with a README or api docs we'll get even more users.

**Comment by DartBot** [comment]
<a href="https://github.com/nex3"><img src="https://avatars.githubusercontent.com/u/188?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [nex3](https://github.com/nex3)**

---

This is one of several packages that have extensive documentation on their primary library but no README. Since only the README is currently visible on pub.dartlang.org (see issue dart-lang/pub-dartlang#294), none of their docs are visible.

I'll move the primary library docs into READMEs for all these packages.

---

_Set owner to @nex3._
_Added [Started](../labels/Started) label._

**Comment by DartBot** [comment]
<a href="https://github.com/nex3"><img src="https://avatars.githubusercontent.com/u/188?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [nex3](https://github.com/nex3)**

---

I'm holding off on this until the new DartDoc viewer supports re-exported libraries, so that the class names in the READMEs have something to link to.

---

_Added [Triaged](../labels/Triaged) label._
_Marked this as being blocked by dart-lang/sdk#12759._

**Comment by DartBot** [comment]
<a href="https://github.com/sethladd"><img src="https://avatars.githubusercontent.com/u/5479?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [sethladd](https://github.com/sethladd)**

---

_Added [Duplicate](../labels/Duplicate) label._
_Marked as being merged into #15._

**Comment by DartBot** [comment]
<a href="https://github.com/anders-sandholm"><img src="https://avatars.githubusercontent.com/u/3276024?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [anders-sandholm](https://github.com/anders-sandholm)**

---

_Removed [Pkg-http](../labels/Pkg-http) label._
_Added [Pkg-HttpServer](../labels/Pkg-HttpServer) label._
